src/components/TheHeader.vue

```diff
@@ -46,3 +46,14 @@ import { useDisplay } from 'vuetify';
 const { mobile } = useDisplay();
 const pageLinks = getPageLinks();
 </script>
+
+<style scoped>
+.tab-with-margin .v-tab {
+  display: flex;
+  align-items: center;
+}
+
+.tab-with-margin .v-tab .v-icon {
+  margin-right: 4px;
+}
```

Comment:
How does this look with and without the center-aligned flex?

My thought was just to put the margin directly in the `<v-icon>` html with the vuetify helper class like

```

            <v-icon
              :icon="`mdi-${icon}`"
              class="mr-1"
              :aria-label="text"
              aria-hidden="false"
              role="img"
            />
```

Reply:
okay Give me some time time and I'll reach you surely